packages/ui/src/views/user-management/components/dialogs/hooks/use-dialog-handlers.ts

```diff
@@ -0,0 +1,46 @@
+import { DialogLabels, IDialogHandlers } from '@/views/user-management/components/dialogs'
+import { useDialogs } from '@/views/user-management/providers/dialogs-provider'
+
+export const useDialogHandlers = (handlers: IDialogHandlers) => {
+  const { closeDialog } = useDialogs()
+
+  return {
+    handleCreateUser: async (...args: Parameters<typeof handlers.handleCreateUser>) => {
+      const result = await handlers.handleCreateUser(...args)
```

Comment:
What happens if these handlers fail?

Reply:
in this case, the error object from `reactQuery` will be returned and displayed on the screen